doc/conf_ref.rst

```diff
@@ -199,7 +199,8 @@ configuration file by your distribution to override the DNF defaults.
     :ref:`boolean <boolean-label>`
 
     Should the dnf attempt to automatically verify GPG verification keys using the DNS
-    system. This option requires libunbound to be installed on the client system. This
+    system. This option requires unbound python module (python3-unbound) to be installed
+    on the client system. This
```

Comment:
> ... This option requires **the** unbound python module ...

I'm also very much debating whether to ask you to re-wrap the whole paragraph... I'm leaning towards "yes", but not sure...

Reply:
I wanted to have minimal diff. But when you request a re-wrap I will do it.